File: Python/1051.py
salario = float(input())

if(salario >= 0 and salario <= 2000):
    print("Isento")

elif(salario > 2000 and salario <= 3000):
    print("R$ %.2f" % ((salario - 2000.0) * 0.08))

elif(salario > 3000 and salario <= 4500):
    print("R$ %.2f" % ((((salario - 2000.0)-(salario - 3000.0))
          * 0.08) + (salario - 3000.0) * 0.18))

elif(salario > 4500):
    print("R$ %.2f" % ((((salario - 2000.0)-(salario - 3000.0))
          * 0.08) + (((salario - 3000.0) - (salario - 4500.0)) * 0.18)+((salario - 4500.0) * 0.28)))

File: Javascript/1051.js
var input = `2001`;
let lines = input.split("\n");
let salario = Number(lines[0]);

if (salario >= 0 && salario <= 2000) {
  console.log("Isento");
} else if (salario > 2000 && salario <= 3000) {
  console.log("R$", ((salario - 2000.0) * 0.08).toFixed(2));
} else if (salario > 3000 && salario <= 4500) {
  console.log(
    "R$",
    (
      (salario - 2000.0 - (salario - 3000.0)) * 0.08 +
      (salario - 3000.0) * 0.18
    ).toFixed(2)
  );
} else if (salario > 4500) {
  console.log(
    "R$",
    (
      (salario - 2000.0 - (salario - 3000.0)) * 0.08 +
      (salario - 3000.0 - (salario - 4500.0)) * 0.18 +
      (salario - 4500.0) * 0.28
    ).toFixed(2)
  );
}
